docs/preparing-your-app-for-testflight.md

```diff
@@ -0,0 +1,143 @@
+# Preparing your App for TestFlight
+
+Publishing to the App Store requires some preparation. This guide should help you get started with using Transporter and the App Connect site to upload your app to the App Store for internal testing via TestFlight. This guide will use the example of publishing a MacCatalyst app to the Mac App Store, but the process is similar for iOS and tvOS apps.
+
+There is some preparation involved with preparing your Apple Developer account for publishing, such as preparing an App Identifier and a Provisioning Profile. The MAUI documentation has a great guide for that:
+https://aka.ms/maui-publish-app-store
+
+Also, your project will need to have configuration values set in the .csproj file, the Info.plist file, and the Entitlements.plist file.
+
+## Preparing your Project File (.csproj) for App Store submission
+
+Before you can submit your app to the App Store, you need to make sure that it is correctly configured. This includes:
+
+- Setting the build to reference the correct `Entitlements.plist` in your project (For example, one for Debug and one for Release).
+- Setting the correct `ApplicationId` in your project.
+- Setting the correct `ApplicationVersion` in your project.
+- Setting the correct `CodesignKey` in your project.
+- Setting the correct `PackagingSigningKey` in your project.
+- Setting the correct `CodesignEntitlements` in your project.
+- Setting the correct `CodesignProvision` in your project.
+
+### Info.plist
+
+The `Info.plist` file is used by the build system to configure your app and is already generated by your project. It contains info such as the Bundle Display Name and Bundle Identifier.
```

Comment:
The bundle identifier can either be set with the CFBundleIdentifier key in the Info.plist file, or the ApplicationId in the project file. This sentence seems to indicate that you have to set CFBundleIdentifier in the Info.plist, which isn't correct if ApplicationId is set in the project file (as mentioned in he previous section).

There are a few other configuration values that can either be set in the project file or the Info.plist file - more info about that here: https://github.com/xamarin/xamarin-android/blob/main/Documentation/guides/OneDotNetSingleProject.md